tests/components/auth/test_init.py

```diff
@@ -606,18 +624,34 @@ async def test_ws_delete_all_refresh_tokens(
 
     tokens = result["result"]
 
-    await ws_client.send_json(
-        {
-            "id": 6,
-            "type": "auth/delete_all_refresh_tokens",
-        }
-    )
+    ws_json = {
+        "id": 6,
+        "type": "auth/delete_all_refresh_tokens",
+    }
+    if delete_token_type:
```

Comment:
We want to avoid checks like this.

Reply:
Are you looking for the `**` operator here? (updated)